src/agents/common.jl

```diff
@@ -197,6 +201,14 @@ function HEMData(input_dir::String; year_start::Int=2020, delta_t::Int=4, epsilo
 
     delta_t = ParamScalar("delta_t", delta_t, description="number of hours per representative hour")
 
+    # inputs for stage 2, these should be read from input files
+    "local zones or communities"
+    index_z_local = Dimension("z_local", [:Community1, :Community2, :Community3])
+    "customer types"
+    index_h_local = Dimension("h_local", [:Res_Community1, :Res_Community2, :Res_Community3])
```

Comment:
Again, these should just be `index_z` and `index_h`. And maybe we should go ahead and start a UVM data creation script in the HEMData repo?